src/commands/send.ts

```diff
@@ -68,6 +68,13 @@ export class SendCommand extends WriteCommand {
         const amount: number = cmdHelper.checkAndGetAmount(args?.amount, flags.sato);
         const satoAmount = flags.sato ? amount : toSatoshi(amount);
 
+        const transactionSatoAmount: number = flags[feesIncludedFlagId] ? satoAmount - flags.fee : satoAmount;
+
+        if (transactionSatoAmount <= 0) {
+            this.stop("Insufficient transferred amount to pay transaction fees.");
+            return undefined;
+        }
+
```

Comment:
Shouldn't this be done in `createTransactionStruct` method?
This is more DRY is this test must be done in another command.

Even, this could be done in SDK or Crypto/Builder?